resources/certificate.rb

```diff
@@ -0,0 +1,28 @@
+resource_name :certificate
+default_action :install
+
+property :certfile, String
+property :cert_password, String
+property :keychain, String
+
+action_class do
+  def keychain
+    if property_is_set?(:keychain)
+      new_resource.keychain
+    else
+      ''
+    end
+  end
+end
+
+action :install do
+  test = SecurityCommand.new(new_resource.certfile, keychain)
```

Comment:
I think we can use a better name than `test` here: perhaps `certificate` or `cert`

Reply:
done